fragment.go

```diff
@@ -189,7 +189,7 @@ func (f *Fragment) Open() error {
 func (f *Fragment) openStorage() error {
 	// Create a roaring bitmap to serve as storage for the slice.
 	if f.storage == nil {
-		f.storage = roaring.NewBitmap()
+		f.storage = roaring.NewBitmapBtree()
```

Comment:
For consistency with the other methods, can we rename this to `NewBTreeBitmap` (note also the capitalized `T` in tree)